### DISCUSSION TITLE
Can I use light potentiometers

### DISCUSSION THREAD

**Discussion by Nicoolai** [nguyenquyhy/Flight-Tracker-StreamDeck#342] [Q&A] [answered]
So, I'm playing around with the Icon A5 and it's panel and cabin lights appear to be potentiometers you turn.
No real on/off.
I can't quite seem to make that work.

I can't find potentiometer events, in https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/main/FlightStreamDeck.Logics/KnownEvents.cs and I can't get on/off toggles to work.

Any way I can control panel and cabin lights in this case?
If I use the toggle events, it can turn it off, if I have turned it up manually but not the other way around.

**Comment by rmroc451** [comment, marked as answer]
You'd need mobiflight to work with those potentiometers, as they weren't exposed via the vanilla simconnect api the last time I looked.

That said, I have them wired into a custom switch panel in airmanager with knobster (I know, not this stream deck plugin).  It still uses the mobiflight wasm module to fire events to simconnect.

It is quite hit or miss.

**Reply by rmroc451** [reply]
go look at mobiflight's [events list](https://github.com/MobiFlight/MobiFlight-Connector/blob/6d003a0a69ac9e5c78a5492651f7dabf08e5d71d/MSFS2020-module/mobiflight-event-module/modules/events.txt), which is supported in the streamdeck plugin.

and look for `LIGHT_POTENTIOMETER_SET`.  The left hand side is the event you'd use in a streamdeck streamdeck button, once you install the mobiflight wasm.

You can play around from there :)

**Reply by Nicoolai** [reply]
Will check that out, thanks.
